Clicked/Core/Utils.lua

```diff
@@ -124,6 +124,33 @@ local shapeshiftForms = {
 	[1456] = {}
 }
 
+local classicShapeshiftForms = {
+	WARRIOR = {},
+
+	PALADIN = {},
+
+	HUNTER = {},
+
+	ROGUE = {},
+
+	PRIEST = {},
+
+	SHAMAN = {},
+
+	MAGE = {},
+
+	WARLOCK = {},
+
+	DRUID = {
+		{ 9634, 5487 }, -- Dire Bear Form, Bear Form
+		{ 1066 }, -- Aquatic Form
+		{ 768 }, -- Cat Form
+		{ 783 }, -- Travel Form
+		{ 24858, 33891 }, -- Moonkin Form, Tree of Life Form
+		{ 40120, 33943 } -- Swift Flight Form, Flight Form
```

Comment:
Is there a reason these have to be in their own tables? Would `{ 9634, 5487, 1066, 768, ... }` not work?

Reply:
It's because we have ranked spell on classic, so Bear Form (5487) is replaced by Dire Bear Form (9634) when the character gains this new spell; the same for Flight Form and Swift Flight Form as I understood (but I don't have a character to check for now). About Moonkin Form and Tree of Life Form, you can't have both at the same time, so they share the same form number.